homeassistant/components/onewire/model.py

```diff
@@ -0,0 +1,21 @@
+"""Type definitions for 1-Wire integration."""
+from __future__ import annotations
+
+from typing import TypedDict
+
+
+class DeviceComponentDescription(TypedDict, total=False):
+    """Device component description class."""
+
+    path: str
+    name: str
+    type: str
+    default_disabled: bool
+
+
+class OWServerDeviceDescription(TypedDict, total=False):
```

Comment:
It looks like we always include all three items in the dict?

Reply:
You are right - `DeviceComponentDescription` has optional items but `OWServerDeviceDescription` doesn't